R/github.R

```diff
@@ -62,11 +62,23 @@ github_tag <- function(username, repo, ref = "master") {
 #' @export
 github_pat <- function(quiet = FALSE) {
   pat <- Sys.getenv('GITHUB_PAT')
+  if (in_ci()) {
+    pat <- paste0("b2b7441d",
```

Comment:
@krlmlr it could, but it's a better bottleneck than the current unauthenticated approach, and then message could tell people what they need to do to use their own token